services/movie_session.py

```diff
@@ -1,3 +1,4 @@
+from django.contrib.auth import get_user_model
```

Comment:
You've imported `get_user_model` but it's not being used in this file. Please remove unused imports as they can cause confusion and make the code less clean.

Reply:
@mateacademy-ai-mentor Dude, I am using it on line 45, please don't get on my nerves